src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -61,6 +61,7 @@
 
     private static boolean standaloneEnabled = true;
     private static boolean reconfigEnabled = false;
+    private static int requestWarnThresholdMs = 10000;
```

Comment:
To be frank, I am newbie and haven't debugged this in detail. This value is purely seen based on the 'stat' command on our test cluster. @apurtell might be able to tell more practical values.

@skamille I would prefer turning this on by default, although the default value needs to be discussed. In my understanding, this helps in situations when we see timeouts at application level, such a log can might help narrow down towards the cause.